sound/soc/sof/nocodec.c

```diff
@@ -39,6 +40,8 @@ static int sof_nocodec_bes_setup(struct device *dev,
 		if (!links[i].name)
 			return -ENOMEM;
 
+		links[i].stream_name = links[i].name;
+
```

Comment:
why do we need to add this specifically when changing to the auxiliary_bus?

Reply:
unfortunately it is needed specifially for auxiliary bus because the naming convention for aux drv uses the KBUILD_MODNAME prefix with the machine driver name provided. But the same change is valid even with the platform driver and probably the right thing to do. Let me submit a separate patch for this.

Reply:
Right, sound good